### PR TITLE
add up_generateLsp23Address docs

### PR DESCRIPTION
**Pull request by biancabuzea200**

### CODE REVIEW

**Comment by Hugoo** [comment]
Related to https://github.com/lukso-network/issue-tracker/issues/25

File: docs/standards/rpc-api.md [hunk 1]
```diff
@@ -168,3 +168,80 @@ params: ['0x311611C9A46a192C14Ea993159a0498EDE5578aC'];
 #### Returns
 
 ##### 1. `String` - New controller address, to be added to the profile by the dapp.
+
+### up_generateLsp23Address {#up_generateLsp23Address}
+
+To perform the deployment, the dApp needs the salt and the controllerAddress. The controllerAddress will be injected in the dataKeys and dataValues array, then the salt and the encoded dataKeys and dataValues are sent to the relayer to deploy the profile. Previously, the relayer had to do several consecutive transactions in order to deploy a profile. Now, all those transactions are taking place at the smart contract level, so the relayer only creates one transaction.
```
Comment (asciiman) [inline docs/standards/rpc-api.md:174]: I think a little more context here would be good.

Start with a single sentence that summarizes what this method is good for: Something like: "This method allows deployed addresses to be re-used on other networks and simplifies the deployment." This is the way that I understand this method, but maybe there is another way to say it?

Have a link to the LSP23 standard, for more information.

Say in what case you'd use this method versus `up_import`.

Comment (Esanim) [reply docs/standards/rpc-api.md:174]: This one is a special-case method. It allows the creation of a UP for the browser extension using LSP23 standard, which under the hood will wrap many operations/transactions into one. It simplifies the deployment. 

`up_import` only works for already existing UPs that you want to add to your extension,
and `up_generateLsp23Address` creates a new UP with the help of the browser extension

**Review by Hugoo — APPROVED** [review]

File: docs/standards/rpc-api.md (hunk 1 above)
Comment (asciiman) [reply docs/standards/rpc-api.md]: Nice. It looks great now!

**Pull request merged**